pkgs/development/tools/build-managers/alire/default.nix

```diff
@@ -7,25 +7,24 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "alire";
-  version = "1.2.2";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "alire-project";
     repo = "alire";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-rwNiSXOIIQR1I8wwp1ROVOfEChT6SCa5c6XnTRqekDc=";
+    hash = "sha256-WF7spXwQR04zIGWazUrbCdeLYOzsk8C6G+cfSS6bwdE=";
 
     fetchSubmodules = true;
   };
 
   nativeBuildInputs = [ gprbuild gnat ];
 
-  # on HEAD (roughly 2c4e5a3), alire provides a dev/build.sh script. for now,
-  # just use gprbuild.
   buildPhase = ''
     runHook preBuild
 
-    gprbuild -j$NIX_BUILD_CORES -P alr_env
+    patchShebangs ./dev/build.sh
+    ./dev/build.sh
```

Comment:
[build.sh](https://github.com/alire-project/alire/blob/v2.0.0/dev/build.sh) can be patched to use `-j$NIX_BUILD_CORES` rather than unlimited.
```
gprbuild -j0 -r -p -P `dirname $0`/../alr_env.gpr "$@"
```

Reply:
I actually submitted a PR on the repo for this: alire-project/alire#1651. I'll apply that patch here. Was wondering if that mattered :)